recipe/meta.yaml

```
@@ -115,34 +50,36 @@ outputs:
-        - {{ pin_subpackage('binutils-meta', exact=True) }}  # [unix]
-        - {{ pin_subpackage('c-compiler', exact=True) }}  # [linux]
       run:
-        - {{ compiler('cxx') }}
-        - {{ pin_subpackage('binutils-meta', exact=True) }}  # [unix]
-    script: build-symlinks.sh  # [unix]
+        - {{ compiler('c') }}
+        - binutils             # [linux]
+        - gcc_linux-aarch64    # [linux and aarch64]
+        - gcc_linux-64         # [linux64]
+        - clang_osx-arm64      # [osx and arm64]
+        - clang_osx-64         # [osx and x86_64]
+        - cctools >=949.0.1         # [osx]
+        - ld64 >=530                # [osx]
+        - llvm-openmp               # [osx]
     test:
       commands:
-        - $CXX --help  # [unix]
-{%- for command in commands['cxx-compiler'] %}  # [unix]
-        - {{ command }} --help > /dev/null  # [unix]
-        - echo "Checking {{ command }} resolves to a path containing ${BUILD}"  # [unix]
-  {%- if cxx_compiler == 'gxx' %}  # [unix]
-        - readlink $(which {{ command }}) | grep -q "${BUILD}"  # [unix]
-  {%- elif cxx_compiler == 'clangxx' %}  # [unix]
-        # The prefixed compilers are a symlink on OSX - check they resolve to the same place
-        - "[ \"$(readlink \"$(which {{ command }})\")\" = \"$(readlink \"$(which ${BUILD}-{{ command }})\")\" ]"  # [unix]
-  {%- endif %}  # [unix]
-{%- endfor %}  # [unix]
+        - $CC --help  # [unix]
+        # Linux-specific environment variable checks
+        - echo "C compiler environment variables:" # [linux]
+        - echo $CC # [linux]
+        - test -f "$CC" || echo "$CC not found" # [linux]
+{%- for command in commands['c-compiler'] %}  # [unix and not linux]
+        - {{ command }} --help > /dev/null  # [unix and not linux]
+{%- endfor %}  # [unix and not linux]
+        - echo "Testing Windows c-compiler"  # [win]
+        - if defined CC (echo %CC% exists) ELSE (echo CC not defined)  # [win]
```

Should `CC` be defined on Windows and if so should this test fail rather than print "not defined"?

---

I added an exit 1 on failure.